.github/workflows/codeql.yml

```diff
@@ -0,0 +1,38 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '23 17 * * 5'
```

Comment:
Does this need to run on a schedule?

Reply:
no, that came with the original copy, will remove